package/Dockerfile.windows

```diff
@@ -56,7 +56,7 @@ RUN $URL = 'https://github.com/Microsoft/K8s-Storage-Plugins/releases/download/v
     \
     Write-Host 'Complete.'
 # download wins
-RUN $URL = 'https://github.com/thxCode/wins/releases/download/v0.0.11/wins.exe'; \
+RUN $URL = 'https://github.com/thxCode/wins/releases/download/v0.1.0/wins.exe'; \
```

Comment:
Should this be moved from `https://github.com/thxCode/wins` to `https://github.com/rancher/wins`? And where is this hosted/being built?

Reply:
Yes, but wins is not released yet due to some drone issue. It will be a binary in github release.